intel_pytorch_extension_py/ops/jit.py

```diff
@@ -17,23 +16,28 @@ def script_(obj, optimize=None, _frames_up=0, _rcb=None):
     if core.get_jit_opt() and hasattr(jit_m, '_c'):
         # Disable mix precision in model fusion, since mixed precision cannot
         # bring any benefits for inference, but will lead to loss of accuracy
-        orig_mixed_type = ipex.get_auto_mix_precision()
-        ipex.enable_auto_mix_precision(None)
-        jit_m = wrap_cpp_module(torch._C._jit_pass_fold_convbn(jit_m._c))
-        ipex.enable_auto_mix_precision(orig_mixed_type)
+        if core.get_mix_bf16_fp32():
```

Comment:
can we align the sequence for scrip/trace and disable mix?
i.e. 
in script mode, 1) call script, 2) disable mix, 3) run fold
in trace mode, 1) disable mix, 2) run trace, 3) run fold.

seems second path is better, right?

Reply:
done.

Reply:
LGTM